experimental/check_gpu_ecc/check_gpu_ecc.py

```diff
@@ -9,11 +9,16 @@
 import socket
 import json
 import csv
+from datetime import timedelta
+from datetime import datetime
```

Comment:
Why not combine this into a single import?

Reply:
done